tools/cell_census_builder/tests/test_util.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
+import pytest
+from scipy.sparse import coo_matrix, csr_matrix, triu
+
+from tools.cell_census_builder.util import array_chunker, is_positive_integral, uricat
+
+
+def test_is_positive_integral() -> None:
```

Comment:
To be pedantic, this method should be called `is_nonnegative_integral` since 0 is an accepted value. If that looks better I can make the change.

Reply:
feel free to improve it!

Reply:
If you do this, please also add a test of empty array, e.g.,

> assert not is_nonnegative_integral(np.array([], dtype=np.float32))

and equivalent for sparse.